tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/NewDirectMessageModelsTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using Bot.Builder.Community.Adapters.Twitter.Webhooks.Models.Twitter;
+
+namespace Bot.Builder.Community.Adapters.Twitter.Tests.Webhooks.Models.Twitter
+{
+    [TestClass]
+    [TestCategory("Twitter")]
+    public class NewDirectMessageModelsTests
+    {
+        [TestMethod]
+        public void EventPropertiesShouldBeSetSuccessfully()
+        {
+            var ev = new Event()
```

Comment:
```suggestion
            var event = new Event()
```

Reply:
It doesn't let me use the 'event' name, it is a reserved keyword, any other name in mind?

Reply:
Maybe `twitterEvent`?